doc/releases/release-notes-3.7.rst

```diff
@@ -1363,6 +1366,14 @@ Libraries / Subsystems
     * Fixed an issue with the SMP structure not being packed which would cause a fault on devices
       that do not support unaligned memory accesses.
 
+    * Added :kconfig:option:`CONFIG_MCUMGR_TRANSPORT_BT_DYNAMIC_SVC_REGISTRATION` that allows users
+      to select whether MCUmgr BT service is statically registered at compile time or
+      dynamically at run time.
+
+    * In FS group, the TinyCrypt used for SHA calculation has been replaced with PSA calls.
+
+    * Following deprecated functions have been removed
```

Comment:
Cut sentence? :)

Reply:
Ha. No I have decided to move it in the middle of writing the sentence.

Reply:
Removed